docs/resources/rule_set.md

```diff
@@ -0,0 +1,138 @@
+---
+subcategory: "Security"
+---
+
+# databricks_rule_set Resource
+
+This resource allows you to manage access rules on Databricks account level resources. For convenience we allow accessing this resource through the Databricks account and workspace.
+
+To configure rule sets through the Databricks account, the provider must be configured with `host = "https://accounts.cloud.databricks.com"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using [AAD tokens](https://registry.terraform.io/providers/databricks/databricks/latest/docs#special-configurations-for-azure) on Azure deployments
+
+## Example usage
+
+Rule set management through Databricks workspace:
```

Comment:
Do we really need to do that through workspace as well? What is the benefit of it?

Reply:
In this system, any user should be able to manage rule sets as long as they have sufficient permissions. It is not restricted to account admins only. Since non account admins still do not have good authentication mechanisms, we are exposing the APIs through the workspace where they can use personal access tokens.